### PR TITLE
start of stats

### PR DESCRIPTION
**Pull request by chaelivieira**

### CODE REVIEW

File: express.js [hunk 1]
```diff
@@ -115,12 +116,14 @@ app.get('/login-redirect', (req, res) => {
...
       return;
     }
+
     try{
     // Exchange the auth code for an access token.
     await client.getToken({
       code: req.query.code,
       redirect_uri: `http://localhost:9000/spotify-callback`
     }).then(async results => {
+      console.log(results);
```
Comment (msgyankee) [inline express.js]: This log can get removed, unless being used for further testing
```suggestion
      
```

File: src/script.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const redisClient = redis.createClient();
+const bluebird = require('bluebird');
+bluebird.promisifyAll(redis.RedisClient.prototype);
+bluebird.promisifyAll(redis.Multi.prototype);
+
+
+
+
+
+const script = () => {
```
Comment (msgyankee) [inline src/script.js]: Is this file called anywhere else?

File: express.js [hunk 3]
```diff
@@ -160,8 +162,22 @@ app.get('/login-redirect', (req, res) => {
...
   }
   });
-  
-  
+
+  app.get('/artists/:id',cors(),async (req, res) => {
+    console.log(req.params.id);
+      let accessToken = await redisClient.hgetAsync(`${req.params.id}`, "accesstoken")
+      var result = {};
+      console.log(accessToken);
```
Comment (msgyankee) [inline express.js:191]: same as above
```suggestion
      
```

File: express.js [hunk 4]
```diff
@@ -160,8 +162,22 @@ app.get('/login-redirect', (req, res) => {
     console.log(e);
   }
   });
-  
-  
+
+  app.get('/artists/:id',cors(),async (req, res) => {
+    console.log(req.params.id);
```
Comment (msgyankee) [inline express.js:188]: same as above
```suggestion
    
```

**Review by msgyankee — APPROVED** [review]

**Review by kirstenalice — APPROVED** [review]

**Pull request merged**